### ISSUE TITLE
Custom voices are not supported in colab

### ISSUE THREAD

**Issue by bakharew** [neonbjb/tortoise-tts#23]
Hi, thank you for making and sharing this awesome work.

I've tried to add a custom voice right in the colab, but got this error. 
However, everything worked fine when I ran it locally.

<img width="1544" alt="Screen Shot 2022-05-04 at 14 21 11" src="https://user-images.githubusercontent.com/26136354/166631967-2ae81872-02c6-404c-bed9-7926ba159a26.png">

**Comment by stevenwaterman** [comment]
I don't think this is a general issue with colab, but perhaps an issue with colab + your audio, or something specific to your colab setup. I have had custom voices working in colab exactly as described in the readme.

**Comment by neonbjb** [comment]
That looks a lot like the audio file got corrupted somehow. Maybe it wasn't finished uploading? Can you try again and re-open this (with the attached file) if it is reproducible?

**Comment by bakharew** [comment]
Tried with several files/voices, reuploading them; got the same result. Files attached.

[Archive.zip](https://github.com/neonbjb/tortoise-tts/files/8644629/Archive.zip)

**Comment by neonbjb** [comment]
Re-opening to investigate. I'll get back to you in a day or two.

**Comment by MoAbeds** [comment]
same Problem

**Comment by neonbjb** [comment]
Sorry for the delay. I've figured it out. The scipy that is packaged with colab is old and will fail if you feed it WAV files with corrupt chunks (which one of your files has). More recent versions will simply skip the corrupt chunks.

I've modified the colab to install the latest version of scipy, which "fixes" the issue. In the process of investigating this, I also built a more friendly way to upload custom voices. It is in the cells after the "random voice". Check it out!
